applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -424,8 +425,8 @@ static void stream_qos_set_cb(struct bt_audio_stream *stream)
 		LOG_DBG("Set %s to PD: %d", headsets[channel_index].ch_name, stream->qos->pd);
 
 		if (playing_state) {
-			ret = bt_audio_stream_enable(stream, lc3_preset_sink.codec.meta,
-						     lc3_preset_sink.codec.meta_count);
+		ret = bt_bap_stream_enable(stream, lc3_preset_sink.codec.meta,
```

Comment:
Is this no longer indented correctly, or is this GitHub -trickery again?

Reply:
I have gone through all files and clang-formatted now